spyder/plugins/projects.py

```diff
@@ -277,6 +282,31 @@ def close_project(self):
             self.clear()
             self.restart_consoles()
 
+    def delete_project(self):
+        """
+        Delete current project without deleting the files in the directory

```

Comment:
A single line and a dot at the end please :-)

`"""Delete current project without deleting the files in the directory."""`